simphony_ui/openfoam_wrapper_creation.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+from simphony.engine import openfoam_file_io, openfoam_internal
+from simphony.core.cuba import CUBA
+
+
+def create_openfoam_wrapper(openfoam_settings):
```

Comment:
Ok, time we move into the documentation domain.

We have a format we use for docs. I think it's the "google python". 

It would be like this
```
""" Creates the openfoam wrapper setup from the settings as provided by the model object

Parameters
----------
openfoam_settings : OpenfoamModel
    The traited model describing the openfoam parameters

Returns
-------
openfoam_wrapper : Wrapper
     an instance of the appropriate Wrapper object, properly configured.
"""
```
Pycharm is smart enough to detect this docstring, and will tell you in other code if you are passing the proper types (it will parse the types in the doc)